index.html

```diff
@@ -20,377 +20,273 @@
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul>
+                </div>
+            </div>
           </div>
-         </div>
-
-        <div class="action">
-          <h3>See Project</h3>
         </div>
-
-
-
-
-        
-
-      </section>
-      <section class="projects">
-
-        
-        <h1>Profesional Art Printing Data</h1>
-        <small>A daily selection of privately personalized reads; 
-          no accounts or sign-ups required. 
-          has been the industry's standard
-        </small>
-        <div class="four-tags">
-          <div class="four-tag">
-           <span>Html</span>
+  
+        <div class="card-2">
+          <div class="fourth-card">
+            <div class="fourth-block">
+              <h1>Profesional Art Printing Data More</h1>
+              <p>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. has been the industry's standard
+              </p>
+                <div class="tags-4">
+                  <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul>
+                </div>
+            </div>
           </div>
-          <div class="four-tag2">
-           <span>bootstrap</span>
+          <div class="fifth-card">
+            <div class="fifth-block">
+              <h1>Profesional Art Printing Data More</h1>
+              <p>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. has been the industry's standard
+              </p>
+                <div class="tags-5">
+                  <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul>
+                </div>
+            </div>
           </div>
-          <div class="four-tag3">
-           <span>Ruby</span>
+          <div class="sixth-card">
+            <div class="sixth-block">
+              <h1>Website Portfolio</h1>
+              <p>A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. has been the industry's standard
+              </p>
+                <div class="tags-6">
+                  <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                  </ul>
+                </div>
+            </div>
           </div>
-         </div>
-
-        <div class="action">
-          <h3>See Project</h3>
         </div>
-
-
-
-
-        
-
-      </section>
-
-      
+      </div>
 
     </section>
-    <section class="about">
-      <h1>
-        About Me
-      </h1>
 
-      
-      <span>
-          Hello I’m a software developer! I can help you build a product , 
+    <section class="about">
+      <div class="about-me">
+        <h1>About me</h1>
```

Comment:
- [ ] [Optional] I would suggest you please use `hx` tag here like `h2, h3,h4` because only we use the `h1` tag per page and it represents the main heading of the page.